src/Analysim.Web/Controllers/ProjectController.cs

```diff
@@ -798,8 +798,28 @@ public async Task<IActionResult> DeleteProject([FromRoute] int projectID)
             var deleteProject = await _dbContext.Projects.FindAsync(projectID);
             if (deleteProject == null) return NotFound(new { message = "Project Not Found" });
 
-            // Remove Project
-            _dbContext.Projects.Remove(deleteProject);
+            // Remove all users that follow the project
+            foreach (var user in deleteProject.ProjectUsers)
+            {
+                _dbContext.ProjectUsers.Remove(user);
+            }
+
+            // Delete from Azure
+            var containerClient = _blobServiceClient.GetBlobContainerClient(deleteProject.Name.ToLower());
+            await containerClient.DeleteBlobIfExistsAsync(deleteProject.Name.ToLower());
+
+            if (containerClient == null) return NotFound(new { message = "Project Not Found" });
```

Comment:
yeah but then you are returning without completing the rest of the project deletion steps! This should not be fatal condition; if there's no container, then don't delete it and move on @kiyros